packages/@okta/vuepress-site/docs/guides/authenticators-google-authenticator/main/ios/enrollmentintegrationsteps.md

```diff
@@ -0,0 +1,550 @@
+                        }
+                    }
+                }
+                Section() {
+                    Button("Continue") {
+                        continueSignIn(skip: false)
+                    }
+                    if showSkipButton {
+                        Button("Skip") {
+                            continueSignIn(skip: true)
+                        }
+                    }
+                    Button("Cancel Sign In", role: .destructive) {
+                        // Handle the case of cancelling the sign-in.
+                        currentState = .unknown
+                        showingSignIn = false
+                    }
+                }
+            }
+            .onAppear() {
+                guard let signInController = SignInController.shared else {
+                    // Handle the error
+                    return
+                }
+                if options == nil {
+                    options = signInController.authenticatorOptions()
+                    selectedOption = options?.selectedField
+                }
+
+                let remediations = signInController.currentResponse?.remediations
+                showSkipButton = remediations?.filter({ $0.type == .skip }).first != nil
+            }
+        }
+    }
+
+    func continueSignIn(skip: Bool) {
+        let completion: ((Result<SuccessResultType, LoginError>) -> Void) = { result in
+            isWaiting = false
+            message = nil
+            switch result {
+            case .success(let result):
+                if let token = result.token {
+                    // Succesful sign in
+                    signedIn = true
+                    showingSignIn = false
+                } else if let message = result.message {
+                    // Show a message on this form.
+                    self.message = message.message
+                } else {
+                    // Handle moving to a new type of form.
+                    currentState = SignInController.shared!.currentRemediation!.type
+                }
+
+            case .failure(let error):
+                // Handle the error
+            }
+        }
+
+        let signInController = SignInController.shared?
+        if skip {
+            // Call a utility function for skipping the selection step.
+            signInController.skipAuthenticatorSelection(completion: completion)
+        } else {
+            // Select an authentictor and request the next step.
+            signInController.selectAuthenticator(authenticator: selectedOption,
+                                                 completion: completion)
+        }
+    }
+}
+
+```
+
+These screenshots show selecting Google Authenticator using the form. The first displays the picker with the label and no selected item. The next screen shows the generated list of choices shown after tapping the picker. The final screen shows the picker with Google Authenticator selected.
```

Comment:
```suggestion
These screenshots show how to select Google Authenticator using the form. The first displays the picker with the label and no selected item. The next screen shows the generated list of choices shown after tapping the picker. The final screen shows the picker with Google Authenticator selected.
```

Reply:
The screenshots don't show how to select, they show the different states during the selection: form with a Picker, the picker list that shows after the user taps the picker (hence the "Back" button), and then the original picker with Google Authenticator chosen. The upcoming commit changes the first sentence to:

> These screenshots show the different UI states for the `Picker` as the user selects Google Authenticator.